packages/manager/apps/order-tracking/index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Order tracking App</title>
+    </head>
+    <body style="padding: 3rem" data-ng-app="orderTrackingApp">
```

Comment:
```suggestion
    <body style="padding: 3rem" data-ng-app="orderTrackingApp" data-ng-strict-di>
```
also inline styles should be avoid

Reply:
👍 for `data-ng-strict-di`, for the inline style, as this part of the code is not intended to be scalable, I don't care

Reply:
It's a quick and dirty single page to demo the order-tracking module it's not intended to be maintained. I known inline styles are not recommended but in the case of a standalone app i think it doesn't matter